configuration/ledger.go

```diff
@@ -87,5 +110,16 @@ func NewLedger() Ledger {
 		Exporter: Exporter{
 			ExportLag: 40, // 40 seconds
 		},
+
+		Replica: Replica{
+			Role:              "root",
```

Comment:
why do we need it? and ParentAddress and ParentPubKey

Reply:
Due to the fact that the component is common for different types of nodes, we need to understand what role from the point of view of replication this node should work. And ParentAddress/ParentPubKey will be empty only for root replica node for rest of all we should define who will be parent replica for us.

Reply:
What kind of nodes? We have a heavy-based api